### PR TITLE
Update pre-lambda prototype call and lambda function

### PR DESCRIPTION
**Pull request by elsiel23**
Changes to the pre-lambda call
- Requires project_id, plan_id, priorities, weights in the request.body
- Creates  a new scenario
- Passes scenario_id and user_id to SQS to trigger the lambda function

Changes to the lambda function
- Requires user_id and scenario_id in the queue message body 
- Makes a forsys patchmax call, ignoring the outputs
- TODO: call /forsys/generate_project_areas_from_lambda_output_prototype with static forsys outputs after #740 is submitted

### CODE REVIEW

File: src/planscape/plan/views.py [hunk 1]
```diff
@@ -821,18 +822,23 @@ def get_scores(request: HttpRequest) -> HttpResponse:
...
-# TODO: finalize call logic after testing piping.
-# NOTE: To send a queue message from your local machine, populate AWS credential args.
-# TODO: feed plan_id and scenario_id from user input into the queue message body 
-def queue_forsys_call(request: HttpRequest) -> HttpResponse:
+# NOTE: To send a queue message from your local machine, populate AWS credentials.
+# TODO: Add tests that mock SQS calls
+def queue_forsys_lambda_prototype(request: HttpRequest) -> HttpResponse:
     try:
         user = get_user(request)
+        scenario_id = create_scenario(request)
```
Comment (strangeties) [inline src/planscape/plan/views.py:830]: Should scenario status be set to pending at this step?

**Comment by strangeties** [comment]
Looks fine!

Of note: I just measured and it takes roughly 20s to fetch and reformat all the condition and attribute data from the DB. Do you think it'd be possible to make the DB call from lambda someday in the future?

**Review by strangeties — APPROVED** [review]
lgtm to unblock, but assuming it'll be checked in after #740.

File: src/planscape/plan/views.py (hunk 1 above)
Comment (elsiel23) [reply src/planscape/plan/views.py:830]: i need to instantiate ForsysGenerationRequestParamsFromDb after this is all done, which is why I didn't update the status. During _validate_scenario, it checks that the status is either initialized or failed. is that ok?

**Comment by elsiel23** [comment]
thanks wendy! i can submit this PR first, there's one remaining change that links the whole flow together. 

for your question, i think that's certainly a possibility. we can make a note of it in the hand off docs and see how SIG wants to handle that

**Pull request merged**